src/components/PersonLink/PerosnLink.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
Error: The filename `PerosnLink.tsx` contains a typo. It should be `PersonLink.tsx` to match the component name and maintain consistency.